aptos-move/framework/aptos-framework/sources/configs/gas_schedule.spec.move

```diff
@@ -22,6 +22,7 @@ spec aptos_framework::gas_schedule {
         use aptos_framework::transaction_fee;
         use aptos_framework::staking_config;
 
+        pragma verify_duration_estimate = 300;
```

Comment:
The number has to be large because of the CI test?

Reply:
There was a timeout issue here when I was working on the PR. I will remove this pragma if it is gone.